app/services/vot_parser.rb

```diff
@@ -0,0 +1,68 @@
+class VotParser
+  class VotParseException < StandardError; end
+
+  VotParserResult = Data.define(:identity_proofing, :credential_usage)
+
+  VECTOR_COMPONENT_REGEXP = /\A(?<component>[A-Z])(?<value>[0-9a-z])\z/
+
+  IDENTITY_PROOFING_VECTOR_VALUES = {
+    '0' => :no_identity_proofing,
+    '1' => :identity_proofing_no_biometric,
+    '2' => :identity_proofing_biometric_required,
+  }.freeze
+  CREDENTIAL_USAGE_VECTOR_VALUES = {
+    '0' => :default,
+    '1' => :no_remember_device,
+    '2' => :unphishable_mfa,
+    '3' => :piv_cac_required,
+  }.freeze
+
+  attr_reader :vector_of_trust
+
+  def initialize(vector_of_trust)
+    @vector_of_trust = vector_of_trust
+  end
+
+  def parse
+    result = { identity_proofing: :no_identity_proofing, credential_usage: :default }
```

Comment:
I feel like these might be easier as plain variables instead of hash keys? since there's only two?

```suggestion
    identity_proofing = :no_identity_proofing
    credential_usage = :default
```
but I don't feel that strongly